includes/chart.js

```diff
@@ -336,6 +336,43 @@ function updateEdges(rowID, edgesGroup) {
         // Append the line to the edges group
         edgesGroup.appendChild(line);
 
+        // Add deontic text between second and third nodes
+        if (link.source === 2 && link.target === 3) {
+
+            // Find the row object that corresponds to the current rowID
+            let rowObj = rowValues.find(row => row[0] === rowID);
```

Comment:
Just a remark for later: Rather than passing the rowID, we may want to consider passing the row/statement object itself so we don't have to search for the right one here.